client/src/components/FlightSearch.js

```diff
@@ -1,66 +1,78 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Grid, Paper, TextField, Button, InputLabel } from "@material-ui/core";
 import { Autocomplete } from "@material-ui/lab";
 
 import useStyles from "../styles/FlightSearch";
 
-// Mock Data
-const cities = [
-  { title: "Vancouver" },
-  { title: "Calgary" },
-  { title: "Toronto" },
-  { title: "Bangkok" },
-];
 
 const FlightSearch = () => {
   const classes = useStyles();
-
   const curr = new Date();
   curr.setDate(curr.getDate());
   const date = curr.toISOString().substr(0, 10);
-
-  const [from, setFrom] = useState("Vancouver");
-  const [to, setTo] = useState("Bangkok");
+  const [from, setFrom] = useState("");
+  const [to, setTo] = useState("");
   const [arrival, setArrival] = useState(date);
+  const [cities, setCities] = useState([])
   const [departure, setDeparture] = useState(date);
   const [travellers, setTravellers] = useState(1);
 
+
+  useEffect((e)=>{
+    const getCities = async () =>{
+      try{
+       const cityListResponse = await fetch("/api/cities", {
+         method: "GET",
+         headers: {
+           "Content-Type": "application/json",
+         },
+       });
+       const cityList = await cityListResponse.json();
+       setCities(cityList)
+      } catch(err){
+        console.error()
+      }
+    }
+    getCities()
```

Comment:
We don't want to do this on component start. We are not using the pre-determined cities API that we were using on the explore page. That's just for a curated list of few cities. On this page we want the user to search for flights from anywhere in the world to anywhere else in the world. So we are using `/places/:regionId` which in turn hits up [this API](https://rapidapi.com/skyscanner/api/skyscanner-flight-search?endpoint=5a9c9edde4b084deb4ea6195).
Actually looking at `/places/:regionId` I don't even think `:regionId` is necessary so you can probably remove it

Reply:
Oh I see. I'll effect the changes now.